src/components/NoteElement.jsx

```diff
@@ -0,0 +1,29 @@
+import { RiDeleteBinLine } from "react-icons/ri";
+import "../App.css";
+
+const NoteElements = ({ note, currentNote, deleteNote, setCurrentNoteId }) => {
+  return (
+    <>
+      <div
+        className={`title overflow-hidden w-full cursor-pointer flex justify-between items-center my-2  ${
+          note.id === currentNote.id ? "selected-note bg-[#4A4E74]" : " "
```

Comment:
Since you only have one alternative, it is better to use the logical AND operator in this case 
```suggestion
          note.id === currentNote.id && "selected-note bg-[#4A4E74]" 
```